hr_holiday_notify_employee_manager/README.rst

```diff
@@ -0,0 +1,69 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==================================
+HR Holiday Notify Employee Manager
+==================================
+
+This module extends the functionality of the Leaves App to make possible
```

Comment:
That is not true. You would become a follower of the employee's leave if you are following his/her department and you would receive a email notification depending on how you've configured yourself as follower of the department. If you are just follower of the employe and/or his/her manager you will neither follow the leave or be notified.